lib/membrane/rtp/vad.ex

```diff
@@ -121,6 +121,8 @@ defmodule Membrane.RTP.VAD do
 
   defp handle_if_present(buffer, nil, state), do: {{:ok, buffer: {:output, buffer}}, state}
 
+  @timestamp_limit Bitwise.bsl(1, 32)
```

Comment:
```suggestion
  @timestamp_limit Bitwise.bsl(1, 32) - 1
```

Reply:
I know it was like this before, but it was wrong. We should calculate to which epoch a number belongs using the length of the cycle, not the maximum value. Take a simple example, like cycle 0..3 and try it with `from_which_cycle/3` implementation.